sdk_container/src/third_party/coreos-overlay/profiles/coreos/base/package.accept_keywords

```diff
@@ -27,7 +29,7 @@
 =app-shells/bash-5.2_p26 ~amd64 ~arm64
 
 # No keyword for arm64 yet.
-=coreos-devel/fero-client-0.1.1 **
+~coreos-devel/fero-client-0.1.1 **
```

Comment:
Shouldn't we just add the keyword to fero-client instead?

Reply:
Yeah, I'd say so too, it's our ebuild. On a tangent - we probably we should fork fero-client to flatcar org, because upstream is archived.

Reply:
Moved keyword to the ebuild and deleted this entry from the profile